decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -67,7 +67,10 @@ class Proposal < Proposals::ApplicationRecord
       scope :state_not_published, -> { where(state_published_at: nil) }
       scope :state_published, -> { where.not(state_published_at: nil) }
       scope :except_rejected, -> { not_rejected.or(state_not_published) }
-      scope :except_withdrawn, -> { not_withdrawn }
+
+      scope :withdrawn, -> { where(withdrawn: true) }
+      scope :not_withdrawn, -> { where(withdrawn: false) }
```

Comment:
I'm ok with changing the `except_withdrawn` to `not_withdrawn` as that seems more consistent with the others naming. (We should add an issue to rename the other `except_` scopes)